code/modules/ghostroles/spawner/simplemob/maintdrone.dm

```diff
@@ -1,20 +1,30 @@
 /datum/ghostspawner/simplemob/maintdrone
 	short_name = "maintdrone"
-	name = "Maintenence Drone"
-	desc = "Maintain and Improve the Systems on the Aurora"
+	name = "Maintenance Drone"
+	desc = "Maintain and Improve the Systems on the Aurora."
 	show_on_job_select = FALSE
 	tags = list("Simple Mobs")
 
 	respawn_flag = MINISYNTH //Flag to check for when trying to spawn someone of that type (CREW, ANIMAL, MINISYNTH)
 	jobban_job = "Cyborg"
-	loc_type = GS_LOC_ATOM
 
 	//Vars regarding the mob to use
-	spawn_mob = /mob/living/simple_animal/rat //The mob that should be spawned
+	spawn_mob = /mob/living/silicon/robot/drone //The mob that should be spawned
 
-/datum/ghostspawner/simplemob/maintdrone/cant_see()
+// we fake it here to ensure it pops up and gets added to SSghostroles.spawners, handling of the fabricators is done below
+/datum/ghostspawner/simplemob/maintdrone/select_spawnlocation()
+	return TRUE
+
+/datum/ghostspawner/simplemob/maintdrone/cant_spawn()
```

Comment:
This should probably check `count_drones() < config.max_maint_drones` too? Maybe?